Automattic/wp-calypso
# Plan: Update CTAs visual style on the plan management page

#### Proposed Changes

* This PR changes the CTAs visual style on the plan management page. ref: pebzTe-cs-p2

#### Testing Instructions



* Go to a site on a paid plan.
* Choose `Upgrades > Plans` on the left sidebar and click `Manage plan` on the main panel.
* The CTAs below the plan details card should look as follows:
  <img width="879" alt="screenshot-2022-09-05-at-15 33 30" src="https://user-images.githubusercontent.com/212034/188624257-ef0fdc53-0151-4970-8e9d-3305de9be434.png">


#### Pre-merge Checklist






Related to #
